src/org/usfirst/frc/team125/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,96 @@
+package org.usfirst.frc.team125.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import edu.wpi.first.wpilibj.*;
+import org.usfirst.frc.team125.robot.RobotMap;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc.team125.robot.util.DebouncedBoolean;
+
+public class Intake extends Subsystem {
+	
+	//intake motors
+	private TalonSRX intakeL = new TalonSRX(RobotMap.INTAKE_LEFT);
+	private TalonSRX intakeR = new TalonSRX(RobotMap.INTAKE_RIGHT);
+	//switch to TalonSRX  when using protoBoard
+	
+	private DoubleSolenoid intakePiston = new DoubleSolenoid(RobotMap.INTAKE_RETRACT_FORWARD, RobotMap.INTAKE_RETRACT_REVERSE);
+	private Solenoid clamp = new Solenoid(RobotMap.INTAKE_CLAMP);
+
+	private DigitalInput smartIntake = new DigitalInput(RobotMap.INTAKE_LIMIT_SWITCH);
+	private static final double minimumSmartIntakeTime = 2.0; // Is 2 seconds too long???
+	private DebouncedBoolean smartIntakeDebouncer = new DebouncedBoolean(minimumSmartIntakeTime);
+
+
+
+	public static final double INTAKE_POWER = 1.0;
+
+	public Intake() {
+		
+		//Left side
+		this.intakeL.configPeakOutputForward(1.0, 0);
+		this.intakeL.configPeakOutputReverse(-1.0, 0);
+		this.intakeL.configNominalOutputForward(0.0, 0);
+		this.intakeL.configNominalOutputReverse(0.0, 0);
+		
+		//Right side
+		this.intakeR.configPeakOutputForward(1.0, 0);
+		this.intakeR.configPeakOutputReverse(-1.0, 0);
+		this.intakeR.configNominalOutputForward(0.0, 0);
+		this.intakeR.configNominalOutputReverse(0.0, 0);
+
```

Comment:
After testing we know the two sides will likely have different peak outputs. Can you make the peak output values class variables to make it easier to experimentally find and implement them?

Reply:
+1

Reply:
They're doing it mechanically they want to still give the motor full power

Reply:
Sounds good, your fix looks good if that's going to be the plan,